### PR TITLE
Reorganize Webhook payload docs and add schema enumerations

### PR DESCRIPTION
**Pull request by spasiu**
- Separated core API into individual files.
- Moved webhook crud to a single 2nd level menu item (manage webhooks).
- Removed Webhooks payloads section and separated into multiple events which include payload schemas.
- Also add schema enumeration to:
  - core > appUser reference
  - core > conversations > menu items
  - core > conversations > action buttons
  - managed accounts > apps

### CODE REVIEW

**Comment by spasiu** [comment]
I guess this should have been two PRs one to cut up the files and the other to make changes... Should I separate them?

File: slate/source/includes/rest/_managed.md [hunk 1]
```diff
@@ -17,6 +17,26 @@ All of the existing core APIs such as `/v1/appusers` and `/v1/webhooks` are acce
 
 Endpoints used for provisioning Smooch apps.
 
+## Schema
+
+The schema describes the fields you can expect to be associated with an app.
+
+### app schema
```
Comment (alavers) [inline slate/source/includes/rest/_managed.md]: Should these headings be capitalized?

File: slate/source/includes/rest/core/_app_users.md [hunk 2]
```diff
@@ -0,0 +1,547 @@
+# App User
+
+The app user object represents an end user using your app. The app user document contains basic profile information such as `givenName`, `surname`, and `email`, as well as any custom user properties you choose to configure.
+
+The `/v1/appusers` path gives you APIs that can be used to update the user's properties, retrieve conversation history, post a message, and track app user events.
+
+## Schema
+
+The schema describes the fields you can expect to be associated with an appUser.
```
Comment (alavers) [inline slate/source/includes/rest/core/_app_users.md]: I don't think this Schema header or subtext belong here. Certainly not if you end up a structure like this:

App User
  Schema
    appUser schema

**Comment by alavers** [comment]
Yeah if you can please separate them :)

**Pull request closed without merging**